client/js/recurring_donations/edit/get-params.js

```diff
@@ -1,23 +1,21 @@
 // License: LGPL-3.0-or-later
-const R = require('ramda')
 
-const splitParam = str =>
-  R.split(/[_;,]/, str)
+const splitParam = str => str.split(/[_;,]/)
 
 module.exports = params => {
   const defaultAmts = '10,25,50,100,250,500,1000'
-  // Set defaults
-  const merge = R.merge({
-    custom_amounts: ''
-  })
-  // Preprocess data
-  const evolve = R.evolve({
-    multiple_designations: splitParam
-  , custom_amounts: amts => R.compose(R.map(Number), splitParam)((amts instanceof String ? amts : R.map(x => x/100, amts).join(',')) || defaultAmts)
-  , custom_fields: fields => R.map(f => {
-      const [name, label] = R.map(R.trim, R.split(':', f))
-      return {name, label: label ? label : name}
-    }, R.split(',',  fields))
-  })
-  return R.compose(evolve, merge)(params)
```

Comment:
So, this file is really doing (almost) the same thing as `client/js/nonprofits/donate/get-params.js`. I'd prefer to revert this change here and then work on combining the two together.

Reply:
sounds good, reverted